aten/src/ATen/core/custom_class.cpp

```diff
@@ -45,4 +45,25 @@ void registerCustomClassMethod(std::unique_ptr<jit::Function> fn) {
   customClassMethods().emplace_back(std::move(fn));
 }
 
+jit::Function* getCustomClassMethod(const c10::QualifiedName& name) {
+  for (auto& fn : customClassMethods()) {
+    if (fn->qualname() == name) {
+      return fn.get();
+    }
+  }
+
+  return nullptr;
+}
+
+std::vector<jit::Function*> getCustomClassMethodsWithPrefix(const c10::QualifiedName& prefix) {
+  std::vector<jit::Function*> result;
+  for (auto& fn : customClassMethods()) {
```

Comment:
Prefix scan is linear time regardless of whether we use a `map` or `vector` but I don't think it's important to focus on optimizing this.